Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1663 @@
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+
+        organized_protocols = []
+        if "HTTPS" in protocols:
+            organized_protocols.extend(HTTPS_PROTOCOLS)
+        if "SOCKS" in protocols:
+            organized_protocols.extend(SOCKS_PROTOCOL)
+
+        data = remove_empty_elements(
+            {
+                "identification_profiles": [
+                    {
+                        "profile_name": profile_name,
+                        "description": description,
+                        "status": status,
+                        "order": order,
+                        "members": {
+                            "protocols": organized_protocols,
+                            "proxy_ports": proxy_ports,
+                            "ip": members_by_subnet,
+                            "url_categories": {
+                                "predefined": predefined_url_categories,
+                                "custom": custom_url_categories,
+                            },
+                        },
+                    }
+                ]
+            }
+        )
+
+        return self._http_request(
+            "POST",
+            f"{V3_PREFIX}/web_security/identification_profiles",
+            json_data=data,
+            resp_type="response",
+            ok_codes=[HTTPStatus.NO_CONTENT],
+        )
+
+    def identification_profiles_update(
+        self,
+        profile_name: str,
+        new_profile_name: str | None,
+        status: str | None,
+        description: str | None,
+        order: int | None,
+        protocols: List[str] | None,
+        proxy_ports: List[str] | None,
+        members_by_subnet: List[str] | None,
+        predefined_url_categories: List[str] | None,
+        custom_url_categories: List[str] | None,
+    ) -> Response:
+        """
+        Update identification profile.
+
+        Args:
+            profile_name (str): Identification profile name.
+            new_profile_name (str | None): Identification profile name to update.
+            status (str | None): Status - enable/disable.
+            description (str | None): Description of identification profile.
+            order (int | None): Index of Identification profile in the collection.
+            protocols (List[str] | None): Protocols - HTTPS/SOCKS.
+            proxy_ports (List[str] | None): Proxy ports.
+            members_by_subnet (List[str] | None): Members by subnet.
+            predefined_url_categories (List[str] | None): Predefined URL categories.
+            custom_url_categories (List[str] | None): Custom URL categories.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+
+        organized_protocols = []
+        if protocols and "HTTPS" in protocols:
+            organized_protocols.extend(HTTPS_PROTOCOLS)
+        if protocols and "SOCKS" in protocols:
+            organized_protocols.extend(SOCKS_PROTOCOL)
```

Comment:
This logic should handle in the command function and not in the Client request. ⚠️
It is more relevant to the command function.
In addition, you used this logic in 'identification_profiles_create' function. You should extract this logic to a new function to avoid repetitive code.